docs/usage/self-hosted-configuration.md

```diff
@@ -612,7 +612,22 @@ JSON files will be stored inside the `cacheDir` beside the existing file-based p
 
 ## requireConfig
 
-If this is set to `false`, it means that Renovate won't require a config file such as `renovate.json` to be present in each repository and will run even if one is missing.
+By default, Renovate needs a Renovate config file in each repository where it runs before it will propose any dependency updates.
+
+You can choose any of these settings:
+
+- `"required"` (default): a repository config file must be present
+- `"optional"`: if a config file exists, Renovate will use it when it runs
+- `"ignored"`: config files in the repo will be ignored, and have no effect
+
+This feature is closely related to the `onboarding` config option.
+The combinations of `requireConfig` and `onboarding` are:
+
+|                        | onboarding=true                                                                                                                                         | onboarding=false                                              |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------- |
+| requireConfig=required | An onboarding PR will be created if no config file exists. If the onboarding PR is closed and there's no config file, then the repository is skipped.   | Repository is skipped unless a config file is added manually. |
+| requireConfig=optional | An onboarding PR will be created if no config file exists. If the onboarding PR is closed and there's no config file, the repository will be processed. | Repository is processed regardless of config file presence.   |
+| requireConfig=ignored  | No onboarding PR will be created and repo will be processed while ignoring any config file present.                                                     | Repository is processed, any config file is ignored.          |
```

Comment:
```suggestion
|                        | `onboarding=true`                                                                                                                                         | `onboarding=false`                                              |
| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------- |
| `requireConfig=required` | An onboarding PR will be created if no config file exists. If the onboarding PR is closed and there's no config file, then the repository is skipped.   | Repository is skipped unless a config file is added manually. |
| `requireConfig=optional` | An onboarding PR will be created if no config file exists. If the onboarding PR is closed and there's no config file, the repository will be processed. | Repository is processed regardless of config file presence.   |
| `requireConfig=ignored`  | No onboarding PR will be created and repo will be processed while ignoring any config file present.                                                     | Repository is processed, any config file is ignored.          |
```

Can we try how it looks when we use `monospace` for the settings?

Reply:
Hi @HonkingGoose 
this is how it looks after your last changes
![image](https://user-images.githubusercontent.com/98313426/169002362-60e208af-9617-4601-84aa-ce0d50674dce.png)